Data_Structures_Answers.md

```diff
@@ -2,17 +2,32 @@ Add your answers to the questions below.
 
```

Comment:
Well done, all answers are correct, however, be aware that on questions 3 and 4, the tasks can be done also in O(1)